src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceMethodRef2.java

```diff
@@ -1,10 +1,10 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+//Compilable with Java8
```

Comment:
Look at diff. This line wasn't here before, you added it when resolving the conflicts. We just recently removed this comment from everywhere.
Please remove it.